src/components/CommentList.jsx

```diff
@@ -0,0 +1,32 @@
+import { fetchCommentsByArticleId } from "../utils/fetchFunctions"
+import { useState, useEffect } from "react";
+
+function CommentList({article_id}){
+
+   const [commentList,setCommentList] = useState([])
+
+
+   useEffect(()=>{
+    fetchCommentsByArticleId(article_id)
+    .then((comments)=>{
+        setCommentList(comments)
+    })
+   },[article_id])
+   
+   
```

Comment:
What if there are no comments associated with an article? In that case you should display a message conditionally to let the user know.